car-bridge/src/main.rs

```diff
@@ -66,7 +69,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                     break;
                 };
 
-                spawn(handle_message(chariott.clone(), response_sender.clone(), message));
+                drainage.spawn(handle_message(chariott.clone(), response_sender.clone(), message));
```

Comment:
There's not much point in making this part of the drainage since any responses from the `handle_message` will never get published since this loop is broken by the cancellation token. Ideally, these are aborted too by the cancellation token.

Reply:
Agreed, removed with f586d8da73a7b381b5e33b64dafe85cd6d19ebea.